CourseApp/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace CourseApp
+{
+    public class Program
+    {
+        public static double Mathhh(double x)
+        {
+            double y;

```

Comment:
Declaration and assignment should be on one line?

Reply:
I guess not, @jskonst, why should they?